src/frontend/src/app/pagination/pagination.component.html

```diff
@@ -0,0 +1,11 @@
+ <nav class="pagination is-centered" role="navigation" aria-label="pagination">
+  <!-- <span>{{ getMin() }} to {{ getMax() }} of {{ count }} Records</span>
+  <span>{{ totalPages() }} pages</span> -->
+  <button class="button pagination-previous" (click)="onPrev()" [attr.disabled]="page === 1 || loading">Previous</button>
```

Comment:
debugging code?

Reply:
No, it is not a debugging code. Actually it displays Minimum and Maximum record of current page with total no. of records and pages. If there is need to show this, I will uncomment it otherwise delete this line.